docs/semgrep-secrets/getting-started.md

```diff
@@ -102,6 +102,7 @@ Use filters to narrow down your results. The following criteria are available fo
 | **Validation state**      | Filter by [whether the secret is actively in use or not](#validation). Semgrep Secrets rules include validators, which can check whether the secret is valid for the service with which it is associated. |
 | **Repository visibility** | Filter by whether the repository's [visibility](#repository-visibility) status. |
 | **Secret type**           | Filter by the type of secret, such as **private key**, or the web service that makes use of the secret, such as **Sendgrid** or **Stripe**. |
+| **Component**          | Filter by [Semgrep Assistant component tags](/semgrep-assistant/overview#component-tags). Semgrep Assistant uses AI to categorize a finding based on its function, such as payments, user authentication, and infrastructure. |
```

Comment:
```suggestion
| **Component**          | Filter by [Semgrep Assistant component tags](/semgrep-assistant/overview#component-tags). Semgrep Assistant uses AI to categorize a finding based on its function, such as payments, user authentication, and infrastructure. |
```

@jaksiks Does the second part ("...categorize a finding based on its function, such as payments, user authentication, and infrastructure") apply to Secrets?

Reply:
@khorne3 If we rephrase to a component tag affecting a file rather than a finding then it is clearer.
```suggestion
| **Component**          | Filter by [Semgrep Assistant component tags](/semgrep-assistant/overview#component-tags). Semgrep Assistant uses AI to categorize a file based on its function, such as payments, user authentication, and infrastructure. |
```